### PR TITLE
feat (azure-iot-device) : Trust Bundle

### PR DESCRIPTION
**Pull request by olivakar**
The sample for trust bundle [provision_symmetric_key_trust_bundle_issuance.py](https://github.com/Azure/azure-iot-sdk-python/pull/987/files#diff-a52b25873aa7ac222d70b8d0680842f4af71433b4b71f1517480bb89dbf9d756)

The other sample is a combination of the cert issuance and trust bundle which may not be needed really. If you guys feel it is not very important we can delete [provision_symmetric_key_trust_bundle_and_client_cert_issuance.py](https://github.com/Azure/azure-iot-sdk-python/pull/987/files#diff-aa56e771169a6b0a5270317096c454400bec8f4519b9c694161712001a0dbe3d)

### CODE REVIEW

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+### Prerequisite Steps for DPS , enrollment and CA
+
+These steps must be done for any scenario inside DPS Cert Management.
+
+* A new DPS instance must created in __West Central US__.
+* Create an Enrollment Group or Individual Enrollment in this DPS instance.
+    * Choose to use symmetric key, TPM key or X.509 for this enrollment.
+    * Link an IoT Hub to the enrollment.
+* Create an internal-use test account with one of Microsoft CA partners by emailing _iotcerts@microsoft.com_
```
Comment (BertKleewein) [inline azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md:19]: This says "internal only". Is this step valid for people outside of Microsoft?

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    * Additionally, request them to add the DPS Service Endpoint to the allow list. `What allow list?`
```
Comment (BertKleewein) [inline azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md:20]: >`What allow list?` [](http://example.com/codeflow?start=84&length=18)

Is this a comment to yourself or is this the name of something?

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* Once the CA account is created there are 2 pieces of that will be provided.
+    * `api_key` -  DigiCert API key AND `profile_id` - DigiCert Client Cert Profile ID.
+* Use DPS Service API to associate the CA object above to the DPS. 
+    ```bash
+    curl -k -L -i -X PUT https://<dps_service_endpoint>/certificateAuthorities/<ca_name>?api-version=2021-11-01-preview -H 'Authorization: <service_api_sas_token>' -H 'Content-Type: application/json' -H 'Content-Encoding: utf-8' -d'{"certificateAuthorityType":"DigiCertCertificateAuthority","apiKey":"<api_key>","profileName":"<profile_id>"}'
+    ```
+   where,
+    * `dps_service_endpoint` - available in overview blade.
```
Comment (BertKleewein) [inline azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md:28]: >blade [](http://example.com/codeflow?start=53&length=5)

Does the reader know what a "blade" is and where to find it?

**Review by BertKleewein — APPROVED** [review]
:shipit:

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md (hunk 1 above)
Comment (olivakar) [reply azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md:19]: No not right now. This document will not be valid once this entire feature goes to public (service + device). But half of these things are not really written anywhere and I had to spend some time discovering some of the commands. So it is helpful now.

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md (hunk 2 above)
Comment (olivakar) [reply azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md:20]: No you literally have to mail them and ask this.

File: azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md (hunk 3 above)
Comment (olivakar) [reply azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md]: Modified to say overview blade of dps details

**Pull request merged**